Release_Protocol.md

```diff
@@ -68,6 +72,14 @@ $ git commit -m 'REL: v1.2.0'
 $ git push -u upstream rel/1.2.0
 ```
 
+If the Contributors needed to be updated:
+
+```Shell
+$ git add src/CHANGES.md mkdocs.yml src/99-appendices/01-contributors.md
```

Comment:
Just add this to (what's currently) L70. Adding a file with no changes has no effect.